macros/mediawiki.external.ejs

```diff
@@ -14,7 +14,7 @@ if (/^(http[s]?:\/\/|mailto:)/.test('' + $0)) {
     text = $0.substr($0.indexOf(' ')+1);
   }
 
-  str = '<a class="external" href="' + link + '" title="' + link + '" target="_blank">' + text + '</a>';
+  str = '<a class="external" href="' + link + '" title="' + link + '" target="_blank" rel="noopener noreferrer">' + text + '</a>';
```

Comment:
```suggestion
  str = '<a class="external" href="' + link + '" title="' + link + '>' + text + '</a>';
```

Reply:
I like this, but first I'd like to understand this crazy macro :)

Reply:
Sorry @ExE-Boss but I've forgotten the justification for your suggestion. Why remove the `target="_blank"`? Because if we remove it, what's the point of bothering with the `rel="noopener noreferrer"`?